src/game.py

```diff
@@ -0,0 +1,41 @@
+from src.cli import Cli
+from src.board import Board
+from src.player import Player
+
+class Game():
+
+    def __init__(self, cli=Cli(), board=Board(), player_1=Player(token="X"),
+                 player_2=Player(token="O")):
+        self.cli = cli
+        self.board = board
+        self.player_1 = player_1
+        self.player_2 = player_2
+
+    def start(self):
+        self.cli.welcome()
+        self.play()
+
+    def play(self):
+        self.cli.log(self.board.stringify_spaces())
+        while not self.board.full():
+            self.turn()
+
+        self.cli.goodbye()
+    
+    def turn(self):
+        player_choice = self.cli.prompt_user()
+        move = self.input_to_index(player_choice)
+
+        self.board.update(move, self.current_player().token)
+        self.cli.log(self.board.stringify_spaces())
+       
+    def input_to_index(self, user_input):
+        input_int = int(user_input)
+        return input_int - 1 
```

Comment:
What does the variable assignment add? Is there a reason you didn't just one-line this to `return int(user_input) - 1`?

Reply:
I was going for clarity, but it sounds like it may be excessive?

Reply:
Yeah, my personal opinion is that it is noise. input_int doesn't add anything that int(user_input) doesn't convey.